mantidproject/mantid
# BASIS: error when reducing multiple files in individual mode

**Description of work.**
- [x] Corrected bug for casting run numbers to strings
- [x] Corrected bug for `Convertunits` to being able to read output `RebinnedOuput` from SofQW3
- [x] System test requesting a dynamic susceptibility workspace
- [x] release notes not needed because it is a bug, not any improvement in functionality

**Report to:**  Naresh Osti

**To test:**
Verify the following script runs with no exceptions
```
from mantid.simpleapi import BASISReduction
kw = dict(RunNumbers='91944-91945',
          EnergyBins=[-120, 0.4, 520.0],
          MomentumTransferBins=[0.3, 0.2, 1.9],
          DivideByVanadium=True,
          NormRunNumbers='90181',
          DoFluxNormalization=True,
          ReflectionType='silicon_111',
          RemoveTemporaryWorkspaces=True,
          DoIndividual=True,
          OutputSusceptibility=True)
BASISReduction(**kw)
```

Fixes #25988

This does not require release notes because it is a small bug correction

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
